native/helloworld-rs/README.md

```diff
@@ -1,27 +1,35 @@
-# Simple "Hello, world!" Rust Program on Unikraft
+# Rust "Hello, World!"
 
-This directory contains an example Rust-based "Hello, world!" example.
+This directory contains a Rust "Hello, World!" example running on Unikraft.
 
-To run this example, install [Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli) and [the `nightly` Rust toolchain channel through Rustup](https://www.rust-lang.org/tools/install).
+To run this example, install [Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli) and [the `nightly` Rust toolchain channel through `Rustup`](https://www.rust-lang.org/tools/install).
 
-Then, clone this repository and cd into this directory.
+## Build
+
+To build the image, clone this repository and `cd` into this directory.
 You can then build the project with:
 
-```
+```bash
 KRAFTKIT_TARGET=helloworld cargo +nightly build -Z build-std=std,panic_abort --target x86_64-unikraft-linux-musl
 ```
 
 In the above command, `KRAFTKIT_TARGET=helloworld` makes the name of the application known to `kraftld`.
 Rust for Unikraft currently only supports the `qemu/x86_64` target.
 If a `Kraftfile` contains more targets, the correct one can be selected through the `KRAFTKIT_PLAT` and `KRAFTKIT_ARCH` environment variables.
 
-Once built, you can execute the unikernel via:
+## Run
 
+In order to run the locally built image, use `kraft`:
+
+```bash
+kraft run --rm --plat qemu --arch x86_64 .
```

Comment:
I am not sure if this works since this uses Cargo and `kraftld` to build. I don't know if `kraft run` picks this up properly.

Reply:
It works. You should try it, for safety.

Reply:
It works for me as well, should be good